SETUP_ARCH.md

```diff
@@ -0,0 +1,23 @@
+# Snowdrift setup instructions for Arch Linux
+
+Arch has the benefit of being very well-documented, so this document is
+very short!
```

Comment:
I have mixed feelings. I like a light tone, but I think it's better to be actually clever/funny/engaging. This sentence seems more like boasting about Arch. It's also very ironic. I think this case should be removed, but *keep* adding lighthearted stuff and some of it will get through ;)